lms/templates/learner_dashboard/right_panel.underscore

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Can this file be renamed something like sidebar. With RTL ```right_panel``` isn't very meaningful.